backend/app/helpers/json_web_token_helper.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module JsonWebTokenHelper
+  SECRET_KEY = 'super_secret_key'
```

Comment:
🤔 é para ser assim mesmo ou deve vir de uma `VAR`.

Reply:
essa chave ai eh usada para seguranca, caso alguem quira hackear nosso token

Reply:
Tou ligado, mas não deveria ser uma key, melhor não. 👀

Reply:
@leonardobrito , eu to de boas com isso assim, da uma commitada ai